msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -82,9 +82,9 @@ For more details, see [chat resource-specific consent permissions](/graph/permis
     1. [Check your app for added RSC permissions in a team](#check-your-app-for-added-rsc-permissions-in-a-team).
     1. [Check your app for added RSC permissions in a chat](#check-your-app-for-added-rsc-permissions-in-a-chat).
 
-## Configure consent settings in the Azure AD portal
+## Configure consent settings
 
-### Configure group owner consent settings for RSC in a team
+### Configure group owner consent settings for RSC in a team (via Azure AD portal)
```

Comment:
Configure group owner consent settings for RSC in a team using the Azure AD portal

Reply:
Updated.